src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/HearingDocumentService.java

```diff
@@ -115,6 +125,13 @@ private boolean isFastTrackApplication(Pair<CaseDetails, String> pair) {
         return isFastTrackApplication.apply(pair.getLeft().getData());
     }
 
+    private boolean isExpressCase(CaseDetails caseDetails) {
+        return featureToggleService.isExpressPilotEnabled()
+            &&  ExpressCaseParticipation.ENROLLED.getValue().equals(
```

Comment:
You could argue that the logic for determining if the case is in the pilot would be better placed in the ExpressPilotService class. I'd say definitely if you're going to need the logic elsewhere

Reply:
The feature toggle is ok there though